woocommerce/woocommerce
# Add HPOS specific k6 test suite

### Changes proposed in this Pull Request:

This PR modifies some of the `k6` merchant test functions to allow excluding individual tests. The use case here is excluding tests which are not relevant to HPOS performance testing. To that end, the PR adds `hpos-baseline-load.js` which excludes those tests.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. In the `plugins/woocommerce` folder, run `pnpm run env:dev --filter=woocommerce `
2. Run `pnpm run env:performance-init --filter=woocommerce`
3. In the `tests/performance` folder, run `k6 run tests/hpos-baseline-load.js`
4. Run `k6 run tests/wc-baseline-load.js`
5. Both test suites should run successfully.
